### PR TITLE
CI Static check and test files

### PR DESCRIPTION
**Pull request by satya-gorli**
What this PR does / why we need it:
The PR contains files related to Static Checks (goMetalint, goFmt,) and testing (Unit testing, integration testing, e2e test) jobs
Which issue this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged): fixes #
CI files
Special notes for your reviewer:
Please review the CI Files.. to unblock other PRs, GoLint call is not put in Makefile
Release note:
None

### CODE REVIEW

**Comment by leonwanghui** [comment]
Could u enable the ci test in travis.yml?

File: test/integration/integrationtest.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    echo "================================================================================================"
+}
+
+
+
+# Start integration test.
+split_line "Start integration test"
+sudo $OPENSDS_DIR/test/integration/prepare.sh
+go test -v github.com/opensds/opensds/test/integration/... -tags integration
+sudo killall -9 osdslet osdsdock
```
Comment (leonwanghui) [inline test/integration/integrationtest.sh]: Missing `osdsapiserver` daemon here.

Comment (satya-gorli) [reply test/integration/integrationtest.sh]: Updated

**Comment by satya-gorli** [comment]
> Could u enable the ci test in travis.yml?

some of the testcases failing writtern y others ...shall I not include test in .travis.yml ?

**Comment by leonwanghui** [comment]
> > Could u enable the ci test in travis.yml?
> 
> some of the testcases failing writtern y others ...shall I not include test in .travis.yml ?

I think there could be some logic errors in your script, because currently it works well for e2e test and e2ef test in CI envrionment.

**Comment by skdwriting** [comment]
@satya-gorli , can we check and close this?

**Comment by leonwanghui** [comment]
@satya-gorli I will close this PR for now, please be free to reopen if required.

**Pull request closed without merging**